### DISCUSSION TITLE
PrimeNg Tree Select Cyclic JSON issue.

### DISCUSSION THREAD

**Discussion by akashsaxena0915** [primefaces/community#1523] [PrimeNG] [answered]
In prime ng tree select I am getting cyclic json after selecting or deselecting any node. what will be the possible solution for this to avoid cyclic structure?

![image](https://github.com/primefaces/community/assets/55920899/f9f1aa05-8fc8-4a2a-914b-aaf000573798)
![image](https://github.com/primefaces/community/assets/55920899/c7e463a6-aa09-4937-9f17-dc2c6f01d0ed)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Could you please create a stackblitz example so we can investigate and identify the issue?

**Reply by akashsaxena0915** [reply]
Hi,
Created a stackblitz example please check
https://stackblitz.com/edit/kjiyq5?file=src%2Fapp%2Fdemo%2Ftree-select-checkbox-demo.html

**Reply by cetincakiroglu** [reply]
Example is broken could you please update it?

**Reply by akashsaxena0915** [reply]
I have fixed the same url.
